aspnet/migration/rc1-to-rc2.rst

```diff
@@ -0,0 +1,352 @@
+
+.. code-block:: c#  
+
+  {
+    "userSecretsId": "aspnet-WebApplication11-b15b61be-47f8-499c-ac42-9f74b8cdbebe",
+    "compilationOptions": {
+      "emitEntryPoint": true,
+      "preserveCompilationContext": true
+  },
+
+
+You no longer need to reference the Tag Helper package ``Microsoft.AspNet.Mvc.TagHelpers``, which was renamed to ``Microsoft.AspNetCore.Mvc.TagHelpers`` in RC2. The package is now referenced by MVC by default.
+
+Changes in views
+^^^^^^^^^^^^^^^^
+
+Views now support relative paths. 
+
+The Validation Summary Tag Helper has changed. 
+
+RC1:
+
+.. code-block:: html 
+
+  <div asp-validation-summary="All" class="text-danger"></div> 
+
+RC2:
+
+.. code-block:: html
+
+  <div asp-validation-summary="ValidationSummary.All" class="text-danger"></div>
+
+ViewComponents changes
+^^^^^^^^^^^^^^^^^^^^^^
+
+The Sync APIs have been removed.
+
+To reduce ambiguity in ViewComponent method selection, we've modified the selection to only allow exactly one ``Invoke`` or ``InvokeAsync`` per ViewComponent.
+``Component.Render``, ``Component.RenderAsync``, and ``Component.Invoke`` have been removed.
+
+``InvokeAsync`` now takes an anonynmous object instead of separate parameters. To use the view component, call @Component.InvokeAsync("Name of view component", <parameters>) from a view. The parameters will be passed to the ``InvokeAsync`` method. The following example demonstrates the ``InvokeAsync`` method call with two parameters:
+
+.. code-block:: c#  
+
+  // RC1 signature 
+  @Component.InvokeAsync("Test", "MyName", 15)  
+
+  // RC2 signatures
+  @Component.InvokeAsync("Test", new { name = "MyName", age = 15 })
+ 
+  @Component.InvokeAsync("Test", new Dictionary<string, object> { ["name"] = "MyName", ["age"] = 15 })
+
+  @Component.InvokeAsync<TestViewComponent>(new { name = "MyName", age = 15})
+
+Updated controller discovery rules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are changes that simplify controller discovery:
+
+There is a new ``Controller`` attribute that can be used to mark a class and their descendants as controllers.
+Classes whose name doesn't end in ``Controller`` and derive from a base class that ends in ``Controller`` are no longer considered controllers. In this scenario the ``[Controller]`` attribute must be applied to the ``Controller`` class itself or to the base class.
+
+We now consider a type to be a controller if all of the following rules apply:
+
+- The type is a public, concrete, non open generic class.
+- [NonController] is not applied to any type of the hierarchy.
+- The type name ends with ``Controller``, or if the ``[Controller]`` attribute is applied to the type or to one of its ancestors.
+- It's important to note that if ``[NonController]`` is applied anywhere in the type hierarchy the discovery conventions will never consider that type or its descendants to be a controller. ``[NonController]`` takes precedence over ``[Controller]``.
+
+
+DNX commands replaced with .NET Core tools
```

Comment:
This section is covered by dnx to dotnet cli migration doc so maybe it's redundant here?